x/chainlink/client/rest/rest.go

```diff
@@ -18,10 +28,86 @@ func RegisterRoutes(clientCtx client.Context, r *mux.Router) {
 	registerTxHandlers(clientCtx, r)
 }
 
-func registerQueryRoutes(clientCtx client.Context, r *mux.Router) {
-	r.HandleFunc("chainlink/feed/data/{feedId}", listFeedDataHandler(clientCtx)).Methods(MethodGet)
+func registerQueryRoutes(clientCtx client.Context, rtr *mux.Router) {
+	r := clientrest.WithHTTPDeprecationHeaders(rtr)
+
+	r.HandleFunc("/chainlink/feed/data/round/{roundId}/{feedId}", listRoundFeedDataHandler(clientCtx)).Methods(MethodGet) // query feed data by roundId and feedId
```

Comment:
is there a difference with ^ and `/chainlink/feed/data/round/{feedId}/{roundId}`?
if there is, i think the roundId should sit lower in hierarchy than feedID

Reply:
`getRoundData` takes two params, 1. `feedId` which is optional, 2. `roundId` which is mandatory. what the path should be if we switch the order?

Reply:
should it be `/chainlink/feed/data/round/{feedId}/{roundId}`?

Reply:
resolved by FeedViewKeeper spec